tests/test_validations.py

```diff
@@ -32,36 +33,44 @@ class TestValidations(unittest.TestCase):
     def test_load_validations(self, mock_logger):
         load_validations()
         mock_logger.info.assert_any_call("loading validation %s", "simple_validation")
-        self.assertTrue("simple_validation" in VALIDATIONS)
+        for v in VALIDATIONS:
+            if VALIDATIONS[v]["__name__"] == "simple_validation":
+                self.assertTrue(True)
 
     @patch("lifeguard.validations.LIFEGUARD_DIRECTORY", "tests/fixtures")
     @patch("lifeguard.validations.logger")
     def test_execute_validation(self, _mock_logger):
         load_validations()
-        response = VALIDATIONS["simple_validation"]["ref"]()
-        self.assertEqual(response.status, NORMAL)
+        for v in VALIDATIONS:
```

Comment:
These changes invalidate the tests.
Before changes when I remove the production code the tests fail.

https://user-images.githubusercontent.com/136954/121609539-7973c700-ca2a-11eb-92bc-4f02dfdfb2c2.mov

After the changes when I remove the production code the tests pass.

https://user-images.githubusercontent.com/136954/121609557-84c6f280-ca2a-11eb-8378-70a8c0b69c6e.mov

The assert commands must be called.
This problem occurs in all changed tests.